bugbug/repository.py

```diff
@@ -546,22 +570,28 @@ def download_commits(repo_dir, date_from):
 
     print(f"Mining {len(revs)} commits using {processes} processes...")
 
-    CHUNK_SIZE = 256
-    revs_groups = [revs[i : (i + CHUNK_SIZE)] for i in range(0, len(revs), CHUNK_SIZE)]
+    CHUNK_SIZE = int(math.ceil(len(revs) / processes))
```

Comment:
I'd like to do these changes separately, so we can separately evaluate their effect

Reply:
Actually, since it's a small part of the runtime, we can keep it.

Reply:
Since we remove file_** from hg_log, then all the chunks contain almost the same amount of work.
So I think it makes sense to do it here because it's a direct consequence of file_** move and it increases the performance improvement (fyi: 1m39 on my computer with 28 workers for all the log).